test/spec/modules/openxAnalyticsAdapter_spec.js

```diff
@@ -676,42 +676,55 @@ describe('openx analytics adapter', function() {
       auctionId: 'test-auction-id'
     };
 
+    const auctionEnd = {
+      auctionId: 'test-auction-id',
+      timestamp: 1586000000000,
+      auctionEnd: 1586000000100,
+      timeout: 3000,
+      adUnitCodes: [AD_UNIT_CODE],
+    };
+
     const bidWonCloseX = {
       requestId: 'test-closex-request-id',
       adId: 'test-closex-ad-id',
       adUnitCode: AD_UNIT_CODE,
       auctionId: 'test-auction-id'
     };
 
+
     function simulateAuction(events) {
       let highestBid;
 
       events.forEach(event => {
         const [eventType, args] = event;
-        openxAdapter.track({ eventType, args });
         if (eventType === BID_RESPONSE) {
           highestBid = highestBid || args;
           if (highestBid.cpm < args.cpm) {
             highestBid = args;
           }
         }
-      });
 
-      openxAdapter.track({
-        eventType: SLOT_LOADED,
-        args: {
-          slot: {
-            getAdUnitPath: () => {
-              return '/12345678/test_ad_unit';
-            },
-            getSlotElementId: () => {
-              return AD_UNIT_CODE;
-            },
-            getTargeting: sinon
-              .stub()
-              .withArgs('hb_adid')
-              .returns(highestBid ? [highestBid.adId] : [])
-          }
+        if(eventType === SLOT_LOADED) {
```

Comment:
space after if, to stay consistent with the rest?

Perhaps Prebid should just use Prettier to ensure a consistency across all of the JS files.